doni/driver/worker/ironic.py

```diff
@@ -215,6 +215,12 @@ def process(
         state_details: "dict" = None,
     ) -> "WorkerResult.Base":
         hw_props = hardware.properties
+
+        capabilities_list = []
+        for key, value in hw_props.get("baremetal_capabilities").items():
```

Comment:
can do stuff like this in one line, i don't think for comprehensions make it harder to read as they are pretty standard Python practice, but it's discretionary :)

```python
capabilities_string = ",".join([f"{key}:{value}" for key, value in hw_props.get("baremetal_capabilities").items()])
```

Reply:
Tried changing it, but it exceeds line length and looks the same 🤷